### ISSUE TITLE
[Nuget] Rename Ressources?

### ISSUE THREAD

**Issue by Herve-M** [ShelbyZ/UCWA.NET#1]
Hello,

I am using your package and I needed to create a private/internal Nuget package from it but I encounter a problem:
UCWA.Net.Ressources.dll will be never loaded/refered as Nuget treats assemblies that end with .Resources.dll as language resource assemblies.

If you think to use Nuget in the future, I think you need to rename the project or the output name.

Herve-M

**Comment by ShelbyZ** [comment]
I'll look into this to see if there is a good workaround as that seems like a strange limitation that Nuget is enforcing.

**Comment by Herve-M** [comment]
Yes, you can learn more in the `Note` section [here](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#deciding-which-assemblies-to-package)

**Comment by ShelbyZ** [comment]
Would it be easiest if I go through the process of packaging it for nuget?  I have seen the process of manually including assembly files into a package.

**Comment by Herve-M** [comment]
I tried some solution for including this dll and it never run (references, files). The dll is included into the Nuget package but never referenced in the project. 
I didn't use the Powershell solution, because I think it's very heavy just for this problem. 
After I am really new to creating Nuget package.
